src/components/page-details.jsx

```diff
@@ -43,12 +43,14 @@ export default class PageDetails extends React.Component {
 
   componentWillUnmount () {
     window.removeEventListener('keydown', this);
+    window.document.querySelector('title').innerText = 'web-monitoring-ui';
```

Comment:
Ah! We should probably set this to `Scanner` (instead of `web-monitoring-ui`) and update the title tag on the page template: https://github.com/edgi-govdata-archiving/web-monitoring-ui/blob/4f9a73301d3d7dad08033825bca10d55c97b09cf/views/main.html#L8

(For context, if it’s gotten too buried over time: analysts using our tools started referring to this software project as a whole as “Scanner,” hence the `Scanner | <url>` naming that was suggested in #193.)

Reply:
Oh interesting, haha.  Ok I'll go in and make this change.